swig/cpp/src/Struct.cpp

```diff
@@ -105,8 +105,7 @@ uint64_t Data::get_uint64() const {
 
 // Val
 Val::Val(sr_val_t *val, S_Deleter deleter) {
-    if (val == nullptr)
-        throw_exception(SR_ERR_INVAL_ARG);
+    if (val == nullptr) throw_exception(SR_ERR_INVAL_ARG);
```

Comment:
Why are these changes here? They do not appear to follow the coding style conventions as used in the rest of the code base...

Reply:
I was getting compiler warnings. The if statement should either be a one liner, or it should have { }'s around the statement to eliminate the warning.

Which is preferred? 

if (vall == nullptr) throw_exception(SR_ERR_INVAL_ARG);

or 

if (vall == nullptr) {
   throw_exception(SR_ERR_INVAL_ARG);
}

Reply:
After looking at the code some more, most of the existing if () throw_exception lines in the code are one liners, there are a few left that are still two liners without parens. There is nothing in the coding style prohibiting one liners, so I will convert the few remaining instances to one liners to be consistent

Reply:
Actually looking at all of the files it's 50/50 so I will change to two liners with {}